src/Workload/Microsoft.Maui.Sdk/Sdk/BundledVersions.in.targets

```diff
@@ -60,9 +60,6 @@
   </ItemGroup>
 
   <!-- These implicit <PackageReference/> pull dependencies from NuGet transitively -->
-  <ItemGroup Condition=" '$(UseMaui)' == 'true' or '$(UseMauiCore)' == 'true' ">
-    <PackageReference Include="Microsoft.Maui.Extensions" Version="$(MauiVersion)" IsImplicitlyDefined="true" />
-  </ItemGroup>
   <ItemGroup Condition=" '$(UseMaui)' == 'true' or '$(UseMauiCore)' == 'true' or '$(UseMauiEssentials)' == 'true' ">
     <PackageReference Include="Microsoft.Maui.Dependencies" Version="$(MauiVersion)" IsImplicitlyDefined="true" />
   </ItemGroup>
```

Comment:
@mattleibow and @jonathanpeppers I had a question around `$(UseMauiEssentials)=true`, as I am not too familiar with where/how that is used. It seems that projects that set this property to true will now get `Microsoft.Extensions` package references by default through `Microsoft.Maui.Dependencies`, but this may not be any different from how things are set up today? Are there cases where a project could be using `$(UseMauiEssentials)=true` without them using the MAUI workload or also setting `$(UseMaui)` to true?

Reply:
If you did `dotnet new android` and then `UseMauiEssentials=true`, it would be like using Xamarin.Essentials without Xamarin.Forms.

With this change, you'd also get `Microsoft.Extensions` packages, but it seems like that would be OK?

Reply:
Not sure... There is a lot of extra things now I suppose such as the logging, config and more.

I suppose the linker will remove it?

What happens when we switch to library packs and no longer use this nuget?

Reply:
> I suppose the linker will remove it?

It looks like Microsoft.Extensions takes up ~148kb uncompressed in a `dotnet new maui` app. It doesn't look like they are marked as trimmable yet.

But I think if the app doesn't use them at all, they would get dropped? `csc` would exclude them?

> What happens when we switch to library packs and no longer use this nuget?

I think this change actually helps this, as you are getting rid of a pack.